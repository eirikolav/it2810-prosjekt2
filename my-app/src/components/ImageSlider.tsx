import React from 'react'
import './Frame.css';

interface ImageSliderProps {
    imageHandleClick: () => void;
    index: number;
    display: string;
}

interface ImageSliderState {
    images: string[];
    index: number;
}

export default class ImageSlider extends React.Component<any, ImageSliderProps, ImageSliderState> {

    render() {
        return (
            <div>
                <div >
                    <img
                        className="image"
                        src={this.props.display}
                        alt="" />
                </div>
                <button onClick={this.props.imageHandleClick}> Bilde </button>
            </div>
        )
    }

};

