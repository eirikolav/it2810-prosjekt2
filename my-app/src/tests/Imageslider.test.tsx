import React from 'react'
import renderer from 'react-test-renderer'
import ImageSlider from "../components/ImageSlider"

test("Test Header", () => {
    const tree = renderer.create(<ImageSlider />).toJSON()
    expect(tree).toMatchSnapshot()
})