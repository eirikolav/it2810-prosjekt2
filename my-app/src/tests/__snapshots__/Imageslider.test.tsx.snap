// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Header 1`] = `
<div>
  <div>
    <img
      alt=""
      className="image"
    />
  </div>
  <button>
     Bilde 
  </button>
</div>
`;
